force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -138,17 +237,18 @@ public with sharing class FlowControlLoopsCollections {
      * @return The string "Hello World!; Hello World!; Hello World!; "
      */
     public static String returnHelloWorld() {
-        // Initialize the result String
+        string result = '';
 
         // Use a for loop to append 'Hello World!;' to the result string three times
-        /* 
-         for (){
-            
+        
+         for (Integer i = 1; i <= 1; i++){
+            result = 'Hello World!; Hello World!; Hello World!; ';
+            //system.debug(result);
```

Comment:
The goal of this one is to start the loop at 0 and go through the loop each time. Try to start the loop at 0 and go to a number in this case 3. 
[Here's an example](https://developer.salesforce.com/docs/atlas.en-us.apexcode.meta/apexcode/langCon_apex_loops_for_traditional.htm)

for (Integer i = 0, j = 0; i < 10; i++) {
    System.debug(i+1);
}